workspaces/factory-workspace/config/glideinWMS.xml

```diff
@@ -89,41 +89,11 @@
          <monitorgroups>
          </monitorgroups>
       </entry>
-      <entry name="el7ce-workspace.glideinwms.org" enabled="False" gatekeeper="el7ce-workspace.glideinwms.org el7ce-workspace.glideinwms.org:9619" gridtype="condor" verbosity="std" work_dir="AUTO" trust_domain="grid" auth_method="scitoken">
```

Comment:
Leave this there, disabled, in case we want a setup where this works

Reply:
Isn't this good as is? I see `enabled="False"`

Reply:
yes, I was asking just not to remove the lines